packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -253,7 +253,7 @@ export class SpecBuilder {
     return this;
   }
 
-  userListens(query: Query, resumeToken?: string): this {
+  userListens(query: Query, resumeFrom?: string | TestSnapshotVersion): this {
```

Comment:
FWIW, the test above would read super cleanly if you made this
```suggestion
  userListens(query: Query, resumeFrom?: { resumeToken?: string; readTime?: TestSnapshotVersion}}): this {
```

Reply:
Done.